docs/sftt/sftt-api-spec-new.md

```diff
@@ -957,13 +961,80 @@ If the user is not a super admin.
 
 ### Import Reservations
 
-`POST api/v1/protected/import/reservations`
+!!! missing "This route has not been implemented yet"
 
+`POST api/v1/protected/import/reservations`
 
 ## Leaderboard Router
 
-!!! missing "This router has not been implemented yet"
+The leaderboard router is used to get the information displayed on the leaderboard, which is blocks completed per user. It is public because future releases possibly include a public leaderboard page. Since none of the information here is strictly private the consequences of making it public are minimal.
```

Comment:
This is nitpicking but say "It is public so that a future release can include a public leaderboard page" and add a comma after "strictly private" in the next sentence